third_party/rust/Cargo.toml

```diff
@@ -120,20 +162,37 @@ version = "1"
 features = ["full"]
 
 [dependencies.time]
-version = "0.3"
+version = "0.3.5"
 features = ["formatting", "macros", "parsing"]
 
 [dependencies.toml]
 version = "0.5"
 features = ["preserve_order"]
 
+[dependencies.tracing]
+default-features = false
+version = "0.1"
+
+[dependencies.tracing-subscriber]
+default-features = false
+version = "0.2.0"
+features = ["ansi", "fmt", "registry"]
```

Comment:
Should these still be set after 639b5cf3aa28137e427446a51de43b8fc5cf4b67? What about `env-filter`, `fmt`, and `json` from components/skus/browser/rs/lib/Cargo.toml?

Reply:
I believe I regenerated it after that change, but I'll double check. The problem was actually that it was missing features it needed

Reply:
yea I think I missed something here after updating. This should actually be ```"ansi",
    "ansi_term",
    "fmt",
    "registry",
    "sharded-slab",
    "thread_local",``` and I think I need to update the skus Cargo.toml to get those without the other defaults that we don't need